scripts/kubernetesci-presubmit-all-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
This script looks almost the same as what we already have in https://github.com/openshift/odo/blob/master/scripts/openshiftci-e2e-4x-psi-tests.sh 

We should reuse what we already have and not create duplication. If needed the existing script should be generalized or extracted into a function.